content/en/docs/Core concepts/concurrency.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Tuning concurrency"
+date: 2020-11-02
+weight: 3
+---
+
+Baker filters and ouput can be configured with a high concurrency profile (input doesn't support
+parallel processing).
```

Comment:
Not sure what does a high concurrency profile means.
Also saying that input doesn't support parallel processing (I'd use concurrent rather than parallel) is a bit misleading since in inputs there's no record processing per se. Also, inputs can be concurrent if you program them to be, inputs can concurrently fetch record, that's what the KCL input does for example.

Rough suggestion:

Baker allows to tune concurrency at various levels of a pipeline:
 - inputs: baker configuration doesn't expose knobs that are input-specific with respect to concurrent as it highly depends on the input source and how the input is implemented
 - filter: baker runs N concurrent filter chains. N default is ? and can be tuned with `procs`
 - output: baker runs N concurrent output. N default is ? and can be tuned with  `procs`